handlers/handlers.go

```diff
@@ -21,25 +27,78 @@ var reCaptcha = recaptcha.R{
 	Secret: os.Getenv("RECAPTCHA_SECRET_KEY"),
 }
 
+//the function for the minifying
+var m = minify.New()
+
+func compileTemplates(filenames ...string) (*template.Template, error) {
+
+	var tmpl *template.Template
```

Comment:
This code file sometimes names template variables _tmpl_ and sometimes just _t_. For consistency, please set all template variable names to one of those.